ansible/roles/test/files/ptftests/advanced-reboot.py

```diff
@@ -865,18 +907,13 @@ def wait_for_ssh_threads():
         self.log("Data plane works again. Start time: %s" % str(self.no_routing_stop))
         self.log("")
 
-        if self.reboot_type == 'fast-reboot':
-            self.no_cp_replies = self.extract_no_cpu_replies(self.upper_replies)
-
         if self.no_routing_stop - self.no_routing_start > self.limit:
             self.fails['dut'].add("Longest downtime period must be less then %s seconds. It was %s" \
                     % (self.test_params['reboot_limit_in_seconds'], str(self.no_routing_stop - self.no_routing_start)))
         if self.no_routing_stop - self.reboot_start > datetime.timedelta(seconds=self.test_params['graceful_limit']):
             self.fails['dut'].add("%s cycle must be less than graceful limit %s seconds" % (self.reboot_type, self.test_params['graceful_limit']))
-        if self.reboot_type == 'fast-reboot' and self.no_cp_replies < 0.95 * self.nr_vl_pkts:
-            self.fails['dut'].add("Dataplane didn't route to all servers, when control-plane was down: %d vs %d" % (self.no_cp_replies, self.nr_vl_pkts))
 
-        if self.reboot_type == 'warm-reboot':
+        if self.reboot_type == 'warm-reboot' and not self.kvm_test:
```

Comment:
health checks are separate and so why checking for `kvm_test` here?

Reply:
Added `handle_advanced_reboot_health_check_kvm` to separate tests for kvm and real switches.